### PR TITLE
Improved CMSIS USART compatibility

### PR DESCRIPTION
**Pull request by ablaylock**

### CODE REVIEW

**Comment by Hadatko** [comment]
We also need sign Contributor agreements. Could you contact me on dusan.cervenka@nxp.com?

File: erpc_c/transports/uart_cmsis_transport.cpp [hunk 1]
```diff
@@ -74,6 +74,13 @@ erpc_status_t UartTransport::init()
 {
     erpc_status_t status = (*m_uartDrv).Initialize(TransferCallback);
     (*m_uartDrv).PowerControl(ARM_POWER_FULL);
+	
+	/*Configure the USART to 9600 Bits/sec */
```
[the remark this answers is not in the document]
Comment (ablaylock) [reply erpc_c/transports/uart_cmsis_transport.cpp]: I am not sure that Control configuration has any guarantee of functioning before Initialize is called it seems to me that maybe the m_uartDrv should be initialized and then passed to the constructor removing the need for init.

**Comment by Hadatko** [comment]
I updated code as i mentioned. Still need solve init function.

**Comment by MichalPrincNXP** [comment]
It seems this PR is already obsolete and could be closed. @ablaylock , @Hadatko , could you confirm, please? Thanks.

**Comment by MichalPrincNXP** [comment]
Closing as already being obsolete.

**Pull request closed without merging**